robot/nano17.urdf.xacro

```diff
@@ -5,9 +5,12 @@
 	<xacro:include filename="$(find force_torque_sensor)/model/ft_sensor.urdf.xacro"/>
```

Comment:
Since this is an example file, I think it is better to create the file `gamma.urdf.xacro` that instantiate the gamma sensor.